[2025-03-26T07:01:24.939+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-26T07:01:24.944+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: import_user_session_data_from_s3.create_stage manual__2025-03-26T07:01:23.691946+00:00 [queued]>
[2025-03-26T07:01:24.946+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: import_user_session_data_from_s3.create_stage manual__2025-03-26T07:01:23.691946+00:00 [queued]>
[2025-03-26T07:01:24.947+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-26T07:01:24.951+0000] {taskinstance.py:2888} INFO - Executing <Task(SnowflakeOperator): create_stage> on 2025-03-26 07:01:23.691946+00:00
[2025-03-26T07:01:24.953+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=649) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-26T07:01:24.954+0000] {standard_task_runner.py:72} INFO - Started process 652 to run task
[2025-03-26T07:01:24.954+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'import_user_session_data_from_s3', 'create_stage', 'manual__2025-03-26T07:01:23.691946+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/import_user_session_data.py', '--cfg-path', '/tmp/tmpk4roaizv']
[2025-03-26T07:01:24.955+0000] {standard_task_runner.py:105} INFO - Job 13: Subtask create_stage
[2025-03-26T07:01:24.969+0000] {task_command.py:467} INFO - Running <TaskInstance: import_user_session_data_from_s3.create_stage manual__2025-03-26T07:01:23.691946+00:00 [running]> on host 2bee9cd353de
[2025-03-26T07:01:24.991+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='import_user_session_data_from_s3' AIRFLOW_CTX_TASK_ID='create_stage' AIRFLOW_CTX_EXECUTION_DATE='2025-03-26T07:01:23.691946+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-26T07:01:23.691946+00:00'
[2025-03-26T07:01:24.991+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-26T07:01:24.997+0000] {sql.py:266} INFO - Executing: 
    CREATE OR REPLACE STAGE dev.raw.blob_stage
    URL = 's3://s3-geospatial/readonly/'
    FILE_FORMAT = (TYPE = 'CSV', SKIP_HEADER = 1, FIELD_OPTIONALLY_ENCLOSED_BY = '"');
    
[2025-03-26T07:01:25.000+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-26T07:01:25.003+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-26T07:01:25.003+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-26T07:01:25.003+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-26T07:01:25.003+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-26T07:01:25.549+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-26T07:01:25.549+0000] {sql.py:509} INFO - Running statement: CREATE OR REPLACE STAGE dev.raw.blob_stage
    URL = 's3://s3-geospatial/readonly/'
    FILE_FORMAT = (TYPE = 'CSV', SKIP_HEADER = 1, FIELD_OPTIONALLY_ENCLOSED_BY = '"');, parameters: None
[2025-03-26T07:01:25.796+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-26T07:01:25.797+0000] {sql.py:518} INFO - Rows affected: 1
[2025-03-26T07:01:25.797+0000] {snowflake.py:422} INFO - Rows affected: 1
[2025-03-26T07:01:25.798+0000] {snowflake.py:423} INFO - Snowflake query id: 01bb42c5-0004-cc3c-0005-a9f70003117e
[2025-03-26T07:01:25.798+0000] {connection.py:788} INFO - closed
[2025-03-26T07:01:25.845+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2025-03-26T07:01:25.930+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-26T07:01:25.930+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=import_user_session_data_from_s3, task_id=create_stage, run_id=manual__2025-03-26T07:01:23.691946+00:00, execution_date=20250326T070123, start_date=20250326T070124, end_date=20250326T070125
[2025-03-26T07:01:25.974+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-26T07:01:25.983+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1378 AirflowProviderDeprecationWarning: Call to deprecated class SnowflakeOperator. (This class is deprecated. Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`. Also, you can provide `hook_params={'warehouse': <warehouse>, 'database': <database>, 'role': <role>, 'schema': <schema>, 'authenticator': <authenticator>,'session_parameters': <session_parameters>}`.)
[2025-03-26T07:01:25.991+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-26T07:01:25.992+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
